try{
    console.log (a)
    console.log('Executei o arquivo')
    console.log('executei o arquivo e deu erro ')
}catch(e){
    console.log('deu erro ai fechei o arquivo')
}finally
{ console.log('eu sempre sou executado ')

}